apps/web/src/actions/datasets/preview.ts

```diff
@@ -1,8 +1,8 @@
 'use server'
 
+import { disk } from '@latitude-data/core/lib/disk'
 import { DatasetsRepository } from '@latitude-data/core/repositories'
 import { previewDataset } from '@latitude-data/core/services/datasets/preview'
-import disk from '$/lib/disk'
```

Comment:
What's the reason to do this?. For using in jobs? I think if you manage to make it work is fine. But also fine that each app init their own disk singleton

Reply:
we need it in jobs yes